src/xo-app/new-vm/index.js

```diff
@@ -608,6 +594,25 @@ export default class NewVm extends BaseComponent {
     })
   )
 
+  _getCoresPerSocketPossibilities = createSelector(
+    () => this.props.pool,
```

Comment:
The real dependency (input) of this selector is `pool.cpus.cores`, not the pool itself.